src/main/java/mops/persistence/directory/DirectoryBuilder.java

```diff
@@ -47,12 +49,12 @@
      * @return this
      */
     public DirectoryBuilder from(@NonNull Directory directory) {
-        this.id = directory.getId();
-        this.name = directory.getName();
-        this.parentId = directory.getParentId();
-        this.groupOwner = directory.getGroupOwner();
-        this.permissionsId = directory.getPermissionsId();
-        this.creationTime = directory.getCreationTime();
+        id = directory.getId();
```

Comment:
why remove this?

Reply:
I don't know probably happend during merge conflict resolving.

Reply:
please undo it